### DISCUSSION TITLE
Add embed or export functionality for Project Insights charts sharing

### DISCUSSION THREAD

**Discussion by bobeagan** [community/community#38014] [Projects and Issues]
It would be helpful if there was a way to generate an embed snippet to share charts from Projects Insights into other places. This would make it easier to communicate status of progress to non-GH users. Another, less valuable but still useful, option would be to generate a static image export that could be embedded into emails or PDFs for sharing. Right now, screenshots are the only way to share these charts with our non-GH people.

**Comment by david-engelmann** [comment]
Is it something where you'd like a pdf with what is displayed in the insights tab or would you want a csv with the raw contribution info?

**Reply by bobeagan** [reply]
Exporting as a PDF or CSV are other options that I imagine people would find value in. I was specifically just talking about exporting an image of a point-in-time version of the graph so that I could add it to various static reporting methods. Although, my ideal preference is for being able to embed a dynamic version of the graph that can be viewed outside of GitHub in something like Coda/Notion/etc.

**Reply by dpal** [reply]
Exporting a table view or a report into a CSV so one can manage it with spreadsheets would be great!

**Reply by callummccoll** [reply]
CSV please!
